communitiesuk/funding-service-design-fund-application-builder
# FS-4528: Build and Deploy to AWS Environments.

#Change description
Added the required files to enable us to deploy FAB into the AWS environments.

-  Unit tests and other appropriate tests added or updated
-  README and other documentation has been updated / added (if needed)
-  Commit messages are meaningful and follow good commit message guidelines 

How to test
Check workflow for successful development and verify by logging in the AWS console.

Screenshots of UI changes (if applicable)
